extensions/resteasy-reactive/jaxrs-client-reactive/deployment/src/main/java/io/quarkus/jaxrs/client/reactive/deployment/JaxrsClientReactiveProcessor.java

```diff
@@ -486,7 +487,7 @@ A more full example of generated client (with sub-resource) can is at the bottom
                 methodIndex++;
 
                 // finding corresponding jandex method, used by enricher (MicroProfile enricher stores it in a field
-                // to later fill in context with corresponding java.lang.reflect.Method
+                // to later fill in context with corresponding java.lang.reflect.Method)
```

Comment:
You likely want to revert this :)

Reply:
no, it's a closing bracket for the one from one line up

Reply:
Oh right. Good catch :)